packages/tupaia-web/src/features/Map/MapLegend/MapLegend.tsx

```diff
@@ -55,11 +56,15 @@ export const MapLegend = ({ hiddenValues, setValueHidden }: LegendProps) => {
   return (
     <>
       <MobileMapLegend>
-        <LegendComponent />
+        <ErrorBoundary>
```

Comment:
I think that it is probably overkill to have one for the mobile and one for the desktop. Can we just have one around both in stead?